### DISCUSSION TITLE
Flagged issue

### DISCUSSION THREAD

**Discussion by HOSEIN8106** [community/community#39514] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello to all friends. my account has been flagged. if possible, spread this problem so that it can be seen and fixed, thanks.
Issue code #1884662 :))

**Comment by HOSEIN8106** [comment, marked as answer]
[GitHub Support](https://support.github.com/contact/account) can help you.

**Reply by ghost** [reply]
I would like my account unflagged as well. Im doing some important work. That by means of being undone doesn't need to be undone. Please consider helping me...

**Reply by wswggg** [reply]
They can, but they don't. I've been waiting for their reply on my case forever...

**Reply by HOSEIN8106** [reply]
They answer your ticket but it takes time. For me, it took one week.

**Reply by iGulitch** [reply]
For me, it took a few months. So I created a new acc instead. They can't asnwer so simple tickets for so long, it's annoying and unprofessional.

**Reply by Clyde17271** [reply]
For instance, my account is currently frozen. Have you gentlemen ever considered that the support team might be experiencing personnel shortages, and that your specific support team is handling numerous issues and could benefit from funding support through donations? For example, my account remains frozen, as does the team's capacity to assist, due to a lack of funding necessary to update support tickets and perform their duties.

**Comment by sleekwebdesigns** [comment]
Hello to all friends. my account has been flagged. if possible, spread this problem so that it can be seen and fixed

**Comment by ghost** [comment]
My account is flagged, immediately after registration. Need help to fix it. I have raised a ticket also but not support or answer provided from last 5 days.

**Reply by Ghislain1** [reply]
Even me  my account is flagged over 2 Months!!, God bamin!!!

**Comment by MastercaptainGameDev** [comment]
Same issue here, lol.

**Comment by Veld-Khan** [comment]
My account has also been flagged for over a week.

**Comment by SohaSarode** [comment]
My account is flagged, immediately after registration. Need help to fix it. I have raised a ticket also but not support or answer provided from last 5 days.

**Comment by Veld-Khan** [comment]
Totally agree.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by moonbase25** [reply]
Now I think I can see something that might be affecting this issue. It seems that your signature shows that you use ProtonMail - me too. We're both blocked.

**Comment by YossefMohamed** [comment]
My account has been flagged. I didn't break any rules.

**Comment by ranjiththonpunuriprofessional** [comment]
My profile is flagged but I did not break any rules. please get this unflaged

**Comment by otter-computer** [comment]
Hi all, if you're having trouble with a flagged account you'll need to [reach out to GitHub Support](https://support.github.com/contact/account) directly. We aren't able to help with account-specific problems in a public forum like this as it may involve dealing with sensitive information. 

If you're having trouble with your account, or feel it has been flagged by mistake, [open a ticket with GitHub Support](https://support.github.com/contact/account).

**Reply by iGulitch** [reply]
Ha-ha-ha! You are funny! You better reply on time on the tickets, then we wouldn't complain here in the discussions! I've waited 3 months for your reply, I even had to creat a new github acc, and then I heard that my previous one was flagged by occasion. WTF?! I don't need 2 github accs. What do I have to do now? To support both? This is awful!

**Reply by Nora-rad** [reply]
@otter-computer  My account has also been flagged for over 3 months, please get this unflagged. I have also raised a ticket but no support or response has been provided for the last 3 months.

**Comment by cybershujin** [comment]
My account was flagged with no notice, no communication no reason given. I only "found out" because of a 404 error. I tried submitting an appeal for reinstatement but it asked me for the reason (I DON'T KNOW GITHUB YOU TELL ME WHY IT WAS FLAGGED). I submitted anyway and did not even get an acknowledgement that I sent in an appeal. No eta, no communication.
**github this is abysmal you should do better**

**Comment by Nora-rad** [comment]
My account has also been flagged for over 3 months, please get this unflagged. I have also raised a ticket but no support or response has been provided for the last 3 months.

**Comment by ontorbhai0** [comment]
Hello GitHub Support,

My account has been flagged and I am unable to authorize third-party applications.  
I believe this might be a mistake. I only use my account for legitimate purposes such as learning, coding, and hosting my repositories.

Please review my account and remove the flag so I can continue using GitHub features normally.

Thank you for your help and support.

Best regards,  
ontorbhai9

**Comment by Clyde17271** [comment]
🧬 Sometimes, instead of abandoning the ship or getting angry at the Commander, it might be better to recognize and admit that they do not have the ability to complete all the work themselves and need support.🧑‍💻👨‍💻👩‍💻🤖👀👍😘✌️👍😘✌️🧬👀🤖👩‍💻👨‍💻🧑‍💻 There is so much to assist with in the work and projects. Please follow us for further details. Thank you for your patience and understanding. I am Charles Raimondo, the owner of this company. Sorry for any inconvenience or frustration I may have caused. Please contact me via direct email or phone to discuss your issue further, so we can work on a solution and plan for future assistance.

**Comment by devendar0** [comment]
My git gub account is red flagged please help team
